src/dialogs/ADialog.ts

```diff
@@ -2,16 +2,33 @@ import {offset} from '../utils';
 
 abstract class ADialog {
 
-  protected static readonly visiblePopups: HTMLElement[] = [];
+  static readonly visiblePopups: HTMLElement[] = [];
+
+  static removePopup(popup: HTMLElement) {
+    const index = ADialog.visiblePopups.indexOf(popup);
+    if(index > -1) {
+      ADialog.visiblePopups.splice(index, 1);
+      popup.remove();
+    }
+    if(ADialog.visiblePopups.length === 0) {
+      document.removeEventListener('keyup', escKeyListener);
```

Comment:
avoid direct access to document, otherwise the library may not work in popups. prefer `popup.ownerDocument`